sdk-scheme-adapter/v2_1_0/backend/paths/fxTransfersById.yaml

```diff
@@ -25,7 +25,7 @@ patch:
   summary: FX Update Conversion State Notification
   tags:
     - Fx
-  operationId: FxTransfersById
+  operationId: FxTransfersPatchById
```

Comment:
@Ujjwal-Izyane @vijayg10  Please be aware that I've updated the operationId here since they're unique and `FxTransfersById` was already used.